### PR TITLE
Feature/upgrade readme with ie11 polyfill note

### PR DESCRIPTION
**Pull request by raphaellueckl**
Polyfill node should be contained in all readmes.

Also, the scaffolding delivers that information now from scratch.

The scaffolding is already tested.

### CODE REVIEW

File: src/components/10-atoms/link/README.md [hunk 1]
```diff
@@ -131,4 +133,4 @@ If the variant is `icon`, using the attribute `icon`'s string value as icon name
 
 ### onClick
 
-On a react-ified component this can be used as a callback function. It will prevent default 
+On a react-ified component this can be used as a callback function. It will prevent default
```
Comment (markus-walther) [inline src/components/10-atoms/link/README.md]: React-ified

Comment (raphaellueckl) [reply src/components/10-atoms/link/README.md]: Not my change, but I can adapt it.

File: src/components/30-organisms/table/README.md [hunk 2]
```diff
@@ -108,11 +110,10 @@ export default App;
 
 **Note: axa-table does _not_ use ShadowDOM, unlike its sister component axa-table-sortable.**
 
-
 ## Properties
 
-| Attribute           | Details                                                                                   |
-| ------------------- | ----------------------------------------------------------------------------------------- |
+| Attribute           | Details                                                                                                                                                       |
+| ------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------- |
```
Comment (LucaMele) [inline src/components/30-organisms/table/README.md]: here looks like prettier broke something. Please double check markdown

File: src/components/30-organisms/table-sortable/README.md [hunk 3]
```diff
@@ -111,10 +112,10 @@ export default App;
 
 ## Properties
 
-| Attribute           | Details                                                                                   |
-| ------------------- | ----------------------------------------------------------------------------------------- |
+| Attribute           | Details                                                                                                                                                       |
+| ------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------- |
```
[the remark this answers is not in the document]
Comment (NoahLiechti) [reply src/components/30-organisms/table-sortable/README.md]: looks good in my opinion: https://github.com/axa-ch/patterns-library/blob/feature/upgrade-readme-with-ie11-polyfill-note/src/components/30-organisms/table-sortable/README.md

[the remark this answers is not in the document]
Comment (raphaellueckl) [reply src/components/30-organisms/table-sortable/README.md]: @LucaMele Prettier automatically formats tables, which is very neat. It looks ultra nice if you open it in the editor.

File: src/components/30-organisms/table/README.md (hunk 2 above)
Comment (raphaellueckl) [reply src/components/30-organisms/table/README.md]: Looks nice

File: src/components/30-organisms/table-sortable/README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (raphaellueckl) [reply src/components/30-organisms/table-sortable/README.md]: https://raw.githubusercontent.com/axa-ch/patterns-library/2395ae13e49cc86974f66d39ac1345d81d217169/src/components/30-organisms/table-sortable/README.md

https://raw.githubusercontent.com/axa-ch/patterns-library/2395ae13e49cc86974f66d39ac1345d81d217169/src/components/30-organisms/table/README.md

**Pull request merged**